web/libs/editor/src/components/Entity/Entity.jsx

```diff
@@ -212,6 +216,7 @@ export default observer(({ store, annotation }) => {
         <Form
           style={{ marginTop: "0.5em", marginBottom: "0.5em" }}
           onFinish={() => {
+            // this component is not used in the new interface anyway
```

Comment:
more clarification in a comment
```suggestion
            // `normInput` is undefined, but this component is not used in the new interface anyway
```